frontend/app/src/store/history/transactions.ts

```
@@ -162,29 +164,33 @@ export const useTransactionStore = defineStore('history/transactions', () => {
 
   const addTransactionEvent = async (
     event: NewEthTransactionEvent
-  ): Promise<ActionStatus> => {
+  ): Promise<ActionStatus<ValidationErrors | string>> => {
     let success = false;
-    let message = '';
+    let message: ValidationErrors | string = '';
     try {
       await addTransactionEventCaller(event);
       success = true;
     } catch (e: any) {
-      message = e.message;
+      if (e instanceof ApiValidationError) {
+        message = e.getValidationErrors(event);
+      }
```

also here, let's not suppress any errors